apps/video/test/marionette/lib/video_info_view.js

```diff
@@ -0,0 +1,116 @@
+/* global module */
+(function(module) {
+  'use strict';
+
+  /**
+   * @constructor
+   * @param {Marionette.Client} client Marionette client to use.
+   */
+  function VideoInfoView(client) {
+    this.client = client.scope({
+    	searchTimeout: 20000
+  	});
+  }
+
+  module.exports = VideoInfoView;
+
+  /**
+   * @const {Object}
+   */
+  VideoInfoView.Selector = Object.freeze({
+    infoView: '#info-view',
+    infoName: '#info-name',
+    infoLength: '#info-length',
+    infoSize: '#info-size',
+    infoType: '#info-type',
+    infoDate: '#info-date',
+    infoResolution: '#info-resolution',
+    closeButton: '#info-close-button'
+
+  });
+
+  VideoInfoView.prototype = {  
+    
+    client: null,
+
+    /**
+    * @return {String} Currently displayed video name text.
+    */
+  	get displayedName() {
+      return this.client.helper.waitForElement(
```

Comment:
I think it is more intuitive and more efficient to use 'findElement' here. I think it is reasonable to expect the caller (or VideoOptionsView.tapMoreInfo) to ensure the "info view" has been displayed before the VideoInfoView getters are called. Also, it would prevent any timing issues where the caller invokes the info view and then tests for the info view properties.